packages/utils/src/syncpromise.ts

```diff
@@ -1,13 +1,13 @@
 import { isThenable } from './is';
 
-/** JSDoc */
+/** SyncPromise internal states */
 enum States {
-  /** JSDoc */
-  PENDING = 'PENDING',
-  /** JSDoc */
-  RESOLVED = 'RESOLVED',
-  /** JSDoc */
-  REJECTED = 'REJECTED',
+  /** Pending */
```

Comment:
That's getting out of hand :D

Reply:
I can revert this, this was to save a few bytes lol.